Sources/Examples/Echo/main.swift

```diff
@@ -32,11 +32,11 @@ let messageOption = Option("message",
                            default: "Testing 1 2 3",
                            description: "message to send")
 
-func makeClientTLS(enabled: Bool) throws -> GRPCClientConnection.TLSMode {
+func makeClientSSLContext(enabled: Bool) throws -> NIOSSLContext? {
```

Comment:
Remove `enabled` and only call this if it needs to be called?

Would also replace the call site with

```
let tlsConfiguration = ssl ? GRPCClientConnection.TLSConfiguration(try makeClientSSLContext()) : nil
```

Reply:
Ah true, that makes more sense now.